sdk/core/core-util/src/base64.browser.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
```

Comment:
can we avoid this import with stubs like here: https://github.com/Azure/azure-sdk-for-js/blob/b835e0bc706dbc930657c32422e5c265a125f002/sdk/core/core-client/src/base64.browser.ts#L5  ?